src/CMakeLists.txt

```diff
@@ -325,7 +325,7 @@ if(NOT TARGET RapidJSON::RapidJSON)
 endif()
 
 option(WITH_FMT_HEADER_ONLY "use header-only version of fmt library" OFF)
-set(WITH_FMT_VERSION "7.0.0" CACHE
+set(WITH_FMT_VERSION "9.1.0" CACHE
```

Comment:
Is bumping to `9.1.0` intentional? Shouldn't be just `8.1.0` taking into account the commit's description?

Reply:
yeah, it has been bumped to 9.1.0 according https://github.com/ceph/ceph/commit/ff7a7386c1f5d8ce0edb1fb4342951fb5b97e5db. Using 9.1.0 is more safe here, may be there are other differents bewteen 8.1.0 and 9.1.0.

Reply:
please note, the reason why i bumped fmtlib to 9.1.0 is to avoid introducing more changes relying on `FMT_DEPRECATED_OSTREAM`. not to bump the minimum required version of fmtlib to 9.1.0.

Reply:
ok, accutally I don't know the minimum required version is. And I have doubt here, why not alway built the fmt,  there have the soure code src/fmt already? If I update the submoulde fmt, there is alway another question wether should I update the  minimum required version here, it is hard to decide the minimum required version  and the error may happen agian.